textures/src/main/java/com/dumbdogdiner/stickyapi/webapi/minecraftheads/HeadInfo.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.webapi.minecraftheads;
+
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import com.google.common.base.Preconditions;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * Helper object to hold all the stuff returned by the head api
+ */
+@SuppressWarnings("FieldMayBeFinal")
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+class HeadInfo {
```

Comment:
This over what is currently here? Might conflict with the Bukkit internal class...
```suggestion
class HeadMeta {
```